src/audioseal/models.py

```diff
@@ -73,13 +74,17 @@ def message(self, message: torch.Tensor) -> None:
     def get_watermark(
         self,
         x: torch.Tensor,
+        sample_rate: int,
```

Comment:
Shall we provide default value for this (16_000), so is it backward compatible ?